frappe/tests/test_naming.py

```diff
@@ -150,3 +151,26 @@ def test_naming_for_cancelled_and_amended_doc(self):
 		self.assertEqual(amended_doc.name, "{}-CANC-1".format(original_name))
 
 		submittable_doctype.delete()
+
+	def test_determine_consecutive_week_number(self):
```

Comment:
It is really nice to have a test for the utility function.
Let's also add a test for generating actual serial names. For autoname and naming series.
Thanks